### PR TITLE
doc: add release note for v1.121.0

### PR DESCRIPTION
**Pull request by yuwenma**
### Change description

Fixes #

### Tests you have done

- [ ] Run `make ready-pr` to ensure this PR is ready for review.
- [ ] Perform necessary E2E testing for changed resources.

### CODE REVIEW

File: docs/releasenotes/release-1.121.md [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 
-*When resources are promoted from alpha to beta, we (generally) ensure they follow our best practices: use of refs on fields where appropriate,
-output fields from GCP APIs are in `status.observedState.*`
-
-* `PlaceholderKind`
-
-## New Resources:
-
-* Added support for `PlaceholderKind` (v1beta1) resource.
+* `DataformRepository`
```
Comment (justinsb) [inline docs/releasenotes/release-1.121.md:13]: Maybe include " (alpha)" - I assume it's alpha

File: docs/releasenotes/release-1.121.md [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
 
 ## New Fields:
 
-* PlaceholderKind
-  * Added `spec.placeholder` field.
+* BigQueryConnection
+  * Added `status.observedState` field.
```
Comment (justinsb) [inline docs/releasenotes/release-1.121.md:18]: Maybe add something like: 
> Added `status.observedState` field, to group fields relating to the state of the object in GCP ("output" fields).

File: docs/releasenotes/release-1.121.md [hunk 3]
```diff
@@ -1,26 +1,23 @@
...
+
+## Fixes:
 
+* BigTable
+  * When autoscaling is enabled (`spec.cluster[].autoscalingConfig.`), reset `numNodes` (`spec.cluster[].numNodes=2`) to disable manual scaling.
```
Comment (justinsb) [inline docs/releasenotes/release-1.121.md:23]: I would say "When autoscaling is enabled (`spec.cluster[].autoscalingConfig.`), do not use `numNodes` (`spec.cluster[].numNodes`) as that applies only to manual scaling."

File: docs/releasenotes/release-1.121.md (hunk 1 above)
Comment (yuwenma) [reply docs/releasenotes/release-1.121.md]: oh, good point!

File: docs/releasenotes/release-1.121.md (hunk 2 above)
Comment (yuwenma) [reply docs/releasenotes/release-1.121.md]: The `observedState` in  BigQueryConnection is actually a fix to the output-only spec. I saw previous release note does not mention too much details and only categorize this as a new field. I'll move it to "Fixes" and give more context.

File: docs/releasenotes/release-1.121.md (hunk 3 above)
Comment (yuwenma) [reply docs/releasenotes/release-1.121.md]: Thank you. Updated.

**Review by justinsb — APPROVED** [review]

**Pull request merged**